docs/modules/ROOT/pages/auth/setup.adoc

```diff
@@ -133,9 +135,11 @@ For example, you might have a function `decodeJWT` which returns a decoded JWT:
 ----
 const decodedJWT = decodeJWT(encodedJWT)
 
-const server = new ApolloServer({
-    schema: neoSchema.schema,
-    context: { jwt: decodedJWT.payload },
+neoSchema.getSchema().then((schema) => {
```

Comment:
It's not on this line.
There seems to be a now outdated `neoSchema.schema` in 
`docs/modules/ROOT/pages/ogm/private.adoc`
`docs/modules/ROOT/pages/type-definitions/database-mapping.adoc`
`docs/modules/ROOT/pages/type-definitions/custom-directives.adoc`
Not sure we have to update them all, just wanted to point it out.